src/render/dom/index.ts

```diff
@@ -104,9 +105,13 @@ export const motion = /*@__PURE__*/ createMotionProxy(allMotionFeatures)
  *
  * @public
  */
-export function createDomMotionComponent(key: string) {
+export function createDomMotionComponent(key: HTMLElements) {
+    type motionTypes = HTMLMotionComponents & SVGMotionComponents
+    type motionTypesKeys = keyof motionTypes
+    type motionTypesValues = motionTypes[motionTypesKeys]
```

Comment:
`MotionTypes[keyof MotionTypes]` is probably fine

Reply:
Sure it is, changing this